packages/cli/src/commands/stream/commits.ts

```diff
@@ -1,22 +1,23 @@
-import { Command, type CommandFlags } from '../../command.js'
+import { Command, STREAM_ID_ARG, SYNC_ARG, SYNC_OPTIONS_MAP, type CommandFlags } from '../../command.js'
 
-export default class Commits extends Command<CommandFlags, { streamId: string }> {
+export default class Commits extends Command<CommandFlags, { streamId: string, sync: string }> {
```

Comment:
The type of `sync` could be more restricted than `string` as it's an enum of known values, also setting OCLIF's `options` array to have it filter for valid inputs.